azblob/zz_generated_models.go

```diff
@@ -6198,9 +6198,9 @@ type UserDelegationKey struct {
 	// SignedTid - The Azure Active Directory tenant ID in GUID format
 	SignedTid string `xml:"SignedTid"`
 	// SignedStart - The date-time the key is active
-	SignedStart time.Time `xml:"SignedStart"`
+	SignedStart string `xml:"SignedStart"`
 	// SignedExpiry - The date-time the key expires
```

Comment:
If we modify a generated file like this, we should make it a swagger directive and regenerate the generated code.

Reply:
I did not hand-edit this file, this change was the result of running swagger on my `swagger/blob.json` changes. :innocent:

Reply:
Yeah, that's also what I meant. Take a look at the files SDK. We don't have anything similar in the blob SDK because we've not had to do this yeat, but if we were to regenerate the blob SDK on a newer version, your change would go away. A proper swagger directive like defined below would allow it to stick between blob service versions.

https://github.com/Azure/azure-storage-file-go/tree/master/swagger

Reply:
Oh!!!  I did not realize blob.json _itself_ was also generated!  I'm less-new to Swagger than I am to Go, but still pretty new, and didn't know directives were a thing.  Will look into it, thank you for setting me straight. 😅

Reply:
No worries at all!